website/docs/plugins/rq_launcher.md

```diff
@@ -0,0 +1,95 @@
+[![Example application](https://img.shields.io/badge/-Example%20application-informational)](https://github.com/facebookresearch/hydra/tree/master/plugins/hydra_rq_launcher/example)
+[![Plugin source](https://img.shields.io/badge/-Plugin%20source-informational)](https://github.com/facebookresearch/hydra/tree/master/plugins/hydra_rq_launcher)
+
+The RQ Launcher plugin provides a launcher for distributed execution and job queuing based on [Redis Queue (RQ)](https://python-rq.org).
+
+### Installation
+This plugin requires Hydra 1.0 (Release candidate)
+```commandline
+$ pip install hydra-rq-launcher --pre
+```
+Usage of this plugin requires a [Redis server](https://redis.io/topics/quickstart).
+
+Note that RQ does [not support Windows](https://python-rq.org/docs/#limitations).
+
+### Usage
+Once installed, add `hydra/launcher=rq` to your command line. Alternatively, override `hydra/launcher` in your config:
+
+```yaml
+defaults:
+  - hydra/launcher: rq
+```
+
+The configuration packaged with the plugin is defined [here](https://github.com/facebookresearch/hydra/blob/master/plugins/hydra_rq_launcher/hydra_plugins/hydra_rq_launcher/config.py). The default configuratio is as follows:
+
+```yaml
+# @package hydra.launcher
+cls: hydra_plugins.hydra_rq_launcher.rq_launcher.RQLauncher
+params:
+  enqueue:
+    job_timeout: null                  # maximum runtime of the job before it's interrupted and marked as failed (in sec)
+    ttl: null                          # maximum queued time before the job before is discarded (in sec)
+    result_ttl: 864000                 # how long successful jobs and their results are kept (in sec), default: 10 days
+    failure_ttl: 8640000               # specifies how long failed jobs are kept (in sec), default: 100 days
+    at_front: false                    # place job at the front of the queue, instead of the back
+    job_id: null                       # job id, will be overidden automatically by a uuid unless specified explicitly
+    description: null                  # description, will be overidden automatically unless specified explicitly
+  queue: default                       # queue name
+  redis:
+    host: ${env:REDIS_HOST,localhost}  # host address via REDIS_HOST environment variable, default: localhost
+    port: ${env:REDIS_PORT,6379}       # port via REDIS_PORT environment variable, default: 6379
+    db: ${env:REDIS_DB,0}              # database via REDIS_DB environment variable, default: 0
+    password: ${env:REDIS_PASSWORD,}   # password via REDIS_PASSWORD environment variable, default: no password
+    mock: ${env:REDIS_MOCK,False}      # switch to run without redis server in single thread, for testing purposes only
+  stop_after_enqueue: false            # stop after enqueueing by raising custom exception
+  wait_polling: 1.0                    # wait time in seconds when polling results
```

Comment:
config design question:
Why separate the redis and the enqeue parameters?

Reply:
Redis parameters hold connection information for the database. In enqueue parameters are parameters that are passed to `RQ`'s enqueue method (https://python-rq.org/docs/#enqueueing-jobs)

Reply:
If it is well structured you can combine the two structured for config purposes and just have a bit more code when you are initializing the queue or calling enqueue.

your call.